src/Symfony/Component/Messenger/Bridge/Amqp/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Deprecated the `prefetch_count` parameter, it has no effect and will be removed in Symfony 6.0.
  * `AmqpReceiver` implements `QueueReceiverInterface` to fetch messages from a specific set of queues.
+ * Fixed message retry, which caused initial message redelivery to all consumers in setup with multiple active consumers.
```

Comment:
` * Add ability to distinguish retry and delay actions`

Reply:
Updated to your phrasing, thank you